CRM/Utils/System/Standalone.php

```diff
@@ -108,21 +108,29 @@ public function setTitle($title, $pageTitle = NULL) {
    * @inheritDoc
    */
   public function appendBreadCrumb($breadcrumbs) {
-    \Civi::$statics[__CLASS__]['breadcrumb'][] = $breadcrumbs;
+    if (!isset(\Civi::$statics[__CLASS__]['breadcrumb'])) {
+      \Civi::$statics[__CLASS__]['breadcrumb'] = [];
+    }
+    \Civi::$statics[__CLASS__]['breadcrumb'] += $breadcrumbs;
```

Comment:
The `+=` operator works well on associative arrays, but if `$breadcrumbs` has numeric keys then this won't actually work & you'd need `array_merge` instead.